### PR TITLE
pynumad attributes

### PR DESCRIPTION
**Pull request by ecamare**

### CODE REVIEW

File: software_attributes/database/pynumad.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- turbine
+system_type:
+- hawt
+system_location:
+- any
+time_domain: 
+- steady
+- dynamic
+loads: true
+governing_equations:
```
Comment (rafmudaf) [inline software_attributes/database/pynumad.yaml:27]: Just yesterday I changed this field to "tag". See the [docs](https://rafmudaf.github.io/WETOStack/portfolio_analysis/attributes_schema.html) for reference. It's more or less the same, though, so you could just change `governing_equations` to `tags`.

File: software_attributes/database/pynumad.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+has_getting_started_docs: true
+has_examples_tutorials: true
+has_design_intent_statement: true
+readability: 7
+project_coverage: 6
+
+# Testing
+testing_framework: true
+continuous_testing: true
+regression_test_coverage: 70
```
Comment (rafmudaf) [inline software_attributes/database/pynumad.yaml:61]: The `regression_test_coverage` and `unit_test_coverage` fields have been consolidated into one `test_coverage` field.

File: software_attributes/database/pynumad.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+has_cli: false
+# language_interfaces:
+feature_accessibility: 100
+
+# Input / Output
+filetypes:
+- yaml
+- csv
+file_structure_stability: in_development
+downstream_integrations:
```
Comment (rafmudaf) [inline software_attributes/database/pynumad.yaml:78]: `downstream_integrations` is changed to `dependencies` to more accurately reflect the intent here. If pyNuMAD uses results from OpenFAST, then it should be listed here. `downstream_integrations` isn't a valid field anymore.

**Comment by rafmudaf** [comment]
@ecamare I made some changes here for you to review. If it looks good, please let me know and I'll merge.

**Comment by ecamare** [comment]
> @ecamare I made some changes here for you to review. If it looks good, please let me know and I'll merge.

Yes, these look good. Just for clarification, am I handling the outdated labels or you?

**Comment by rafmudaf** [comment]
I've updated all the fields in the latest commit, so it is conforming now.

**Comment by ecamare** [comment]
Okay, thanks. Anything else I need to do regarding the PR?

**Comment by rafmudaf** [comment]
If it all looks accurate to you, then it's ready to roll. I'll go ahead and merge it, and feel free to leave any additional comments if you see anything incorrect.

**Pull request merged**